org.qutebrowser.qutebrowser.yml

```diff
@@ -127,16 +139,14 @@ modules:
       - find /app/share/pdf.js -type f -exec chmod 644 {} \;
     sources:
       - type: file
-        url: https://github.com/mozilla/pdf.js/releases/download/v2.1.266/pdfjs-2.1.266-dist.zip
-        sha256: db57b4d1f16a25427ed5afd51ec09a8bbaa12e42daee9178559097bb1940f643
+        url: https://github.com/mozilla/pdf.js/releases/download/v2.4.456/pdfjs-2.4.456-dist.zip
+        sha256: bd0c36f90130150898890a9972b053601c743ba782543df498a862266f5a7077
 
   - name: qutebrowser
     buildsystem: simple
     build-commands:
       - make --file misc/Makefile install PREFIX=/app
     sources:
       - type: archive
-        url: https://github.com/qutebrowser/qutebrowser/releases/download/v1.7.0/qutebrowser-1.7.0.tar.gz
-        sha256: f1f01b33669a9b08071997bb2642d46a1e5be9e293bdeb817d83db8dc5aad273
-      - type: patch
```

Comment:
Any reason for removing just the patch from here?

Reply:
Two reasons:
- the patch failed to be applied to v1.13.0; and
- at the time, I got to the conclusion that it didn't make any difference (don't ask me why).

Now, I have created a new patch for that, and included it in the yml.